tests/components/accuweather/test_sensor.py

```diff
@@ -0,0 +1,598 @@
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-wetbulbtemperature",
+        suggested_object_id="home_wet_bulb_temperature",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-windchilltemperature",
+        suggested_object_id="home_wind_chill_temperature",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-windgust",
+        suggested_object_id="home_wind_gust",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-cloudcoverday-0",
+        suggested_object_id="home_cloud_cover_day_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-cloudcovernight-0",
+        suggested_object_id="home_cloud_cover_night_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-grass-0",
+        suggested_object_id="home_grass_pollen_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-mold-0",
+        suggested_object_id="home_mold_pollen_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-ozone-0",
+        suggested_object_id="home_ozone_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-ragweed-0",
+        suggested_object_id="home_ragweed_pollen_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-realfeeltemperatureshademax-0",
+        suggested_object_id="home_realfeel_temperature_shade_max_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-realfeeltemperatureshademin-0",
+        suggested_object_id="home_realfeel_temperature_shade_min_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-tree-0",
+        suggested_object_id="home_tree_pollen_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-windgustday-0",
+        suggested_object_id="home_wind_gust_day_0d",
+        disabled_by=None,
+    )
+    registry.async_get_or_create(
+        SENSOR_DOMAIN,
+        DOMAIN,
+        "0123456-windgustnight-0",
+        suggested_object_id="home_wind_gust_night_0d",
+        disabled_by=None,
+    )
+
+    await init_integration(hass, forecast=True)
+
+    state = hass.states.get("sensor.home_apparent_temperature")
+    assert state
+    assert state.state == "22.8"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)